lib/src/main/java/com/airbnb/rxgroups/ObservableGroup.java

```diff
@@ -51,13 +51,8 @@ public long id() {
    * Adds an {@link Observable} and {@link Observer} to this group and subscribes to it. If an
    * {@link Observable} with the same tag is already added, the previous one will be canceled and
    * removed before adding and subscribing to the new one.
-   *
-   * @return a {@link RequestSubscription} reference with which the {@link Observer} can stop
-   * receiving items before the {@link Observable} has completed.
-   * @throws IllegalStateException if an {@link Observable} with the same tag is already added and
-   *                               not yet completed.
    */
-  <T> RequestSubscription add(String tag, Observable<T> observable, Observer<? super T> observer) {
+  <T> void add(String tag, Observable<T> observable, Observer<? super T> observer) {
```

Comment:
this is now only called from `GroupSubscriptionTransformer`, which doesn't use the return value, so it's pretty much pointless to return something here. We could, yeah, but I'd rather to explicitly not to avoid confusion